static/scripts/rewards/render-transaction/read-claim-data-from-url.ts

```diff
@@ -32,7 +33,7 @@ export async function readClaimDataFromUrl(app: AppState) {
   app.claimTxs = await getClaimedTxs(app);
 
   try {
-    app.provider = await useRpcHandler(app);
+    app.provider = await useRpcHandler(app.claims[0]);
```

Comment:
If we have multiple permits with different networks they'd only be able claim those that are on the same network as `claim[0]`

Reply:
I feel like we can implement single network support to start because it seems unusual to me to provide rewards across multiple networks?